pkgs/development/libraries/freeimage/default.nix

```diff
@@ -11,9 +11,40 @@ stdenv.mkDerivation {
     sha256 = "1z9qwi9mlq69d5jipr3v2jika2g0kszqdzilggm99nls5xl7j4zl";
   };
 
-  patches = lib.optional stdenv.isDarwin ./dylib.patch;
+  patches = [
+    # This patch unbundles the source code and removes the vendored dependencies, allowing us to use our own dependencies.
+    # This is vital for fixing AArch64 support and also avoiding CVE-2019-12212 and CVE-2019-12214.
+    (fetchpatch {
+      name = "CVE-2019-12212.CVE-2019-12214.patch";
```

Comment:
I'd call this `unbundle-libs.CVE-2019-12212.CVE-2019-12214.patch`

Reply:
I resolved this